entrypoint.sh.addHostUserToContainer

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# Add local user
+# Either use the LOCAL_USER_ID if passed in at runtime or
+# fallback
+
+USER_ID=${LOCAL_USER_ID:-9001}
```

Comment:
What do you think about RUN_USER_ID ? I assume you set in the run line, or we can do docker-compose.

I think there is a name we can give this script that will make it run at init time.